ga4gh/datarepo.py

```diff
@@ -385,6 +388,11 @@ def _readSystemTable(self, cursor):
         row = cursor.fetchone()
         self._repositoryVersion = config["repositoryVersion"]
         self._creationTimeStamp = config["creationTimeStamp"]
+        if self._repositoryVersion != self.version:
+            raise exceptions.ConfigurationException(
```

Comment:
Is this caught by the CLI, or does it fall through as a stack trace? We should aim to give friendly error messages from now on, and only dump a stack trace in a genuine bug situation.